src/containers/CommunityProjects/CommunityProjects.tsx

```diff
@@ -0,0 +1,34 @@
+import { Page, Card } from "@shopify/polaris";
+import React from "react";
+import { projInfo } from "./CommunityProjectsInfo";
+import "./CommunityProjectsStyles.css";
+
+export function CommunityProjects() {
+  const infoCards: JSX.Element[] = [];
+  projInfo.forEach((element) => {
+    const card = (
+      <div className="singleCardWrapper">
+        <Card
+          title={element.title}
+          sectioned
+          footerActionAlignment="left"
+          primaryFooterAction={{
+            content: "View Project",
+            external: true,
+            url: String(element.link),
+          }}
+        >
+          <p>{element.description}</p>
+        </Card>
+      </div>
+    );
+    infoCards.push(card);
+  });
+
+  return (
+    <Page title="Community Projects">
+      <p>Explore more COVID-19 projects built by people from across Canada!</p>
+      <div className="cardFlexBox">{infoCards}</div>
+    </Page>
+  );
```

Comment:
My suggestion is to just map over `projInfo` directly. I'm not a fan with interim variables and think JSX is well suited to handle these type of scenarios. You could also take it one step further and create a `CommunityProjectCard` component that could encapsulate the logic for rendering one of these cards.

```suggestion
  return (
    <Page title="Community Projects">
      <p>Explore more COVID-19 projects built by people from across Canada!</p>
      <div className="cardFlexBox">
        {projInfo.map((project, index) => (
          <div key={index} className="singleCardWrapper">
            <Card
              title={project.title}
              sectioned
              footerActionAlignment="left"
              primaryFooterAction={{
                content: "View Project",
                external: true,
                url: String(project.link),
              }}
            >
              <p>{project.description}</p>
            </Card>
          </div>
        )}
      </div>
    </Page>
  );
```

Reply:
You're right this looks much better. I went ahead and created a separate component for the individual card as well. https://github.com/Vaccine-Hunters-Canada/VaxFinder-Frontend/pull/45/commits/e65f549e9bfee3d175a7ea42bd8fa2ae43e3477d
https://github.com/Vaccine-Hunters-Canada/VaxFinder-Frontend/pull/45/commits/2240bccb8b2b056e59857a39320a88e34e69bfea